src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] `App.js` should be a class based component, kindly fix this

Reply:
Hi @Munsa1 ! Thank you I will fix it.